quantum/rgblight.c

```diff
@@ -189,6 +189,8 @@ void eeconfig_debug_rgblight(void) {
 }
 
 void rgblight_init(void) {
+    ws2812_init();
```

Comment:
Looks like this needs to be excluded if custom drivers are enabled. 

```suggestion
#ifndef RGBLIGHT_CUSTOM_DRIVER
    ws2812_init();
#endif
```

Otherwise, all of the custom drivers will need to be updated .....